laze-project.yml

```diff
@@ -420,6 +420,7 @@ modules:
   - name: release
     selects:
       - ?lto
+      - ?defmt
```

Comment:
I think this can actually go into the `riot-rs` context, next to `executor-default`.